packages/components/src/components/Link/Link.tsx

```diff
@@ -96,6 +93,7 @@ export type LinkAnalytics = {
   onPress?: () => void
   onConfirm?: () => void
   onCancel?: () => void
+  onOpenURLError?: (e?: unknown, url?: LinkProps['url']) => void
```

Comment:
Is `e` always `string | object`, or can it sometimes be neither?

Reply:
As noted in other comment, I didn't dig exhaustively deeply, but readily [found a StackOverflow discussion](https://stackoverflow.com/questions/54649465/how-to-do-try-catch-and-finally-statements-in-typescript) that seemed to suggest errors can be whatever they (whomever coded the error) want and `unknown` seems to be the recommended typing.